docs/application/dotnet/guides/connectivity/stc.md

```diff
@@ -0,0 +1,166 @@
+# Smart Traffic Control (STC)
```

Comment:
@divyk As a new page is added, please don't forget to guide to update the toc_all.md.

@prasadam-p Please check the application/toc_all.md file and add the link to this file at the corresponding location.

Reply:
@mijin85cho , sure, thanks.

Reply:
done